packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -423,9 +423,6 @@ export default defineComponent({
     expandPeoplesPanel() {
       this.setSidebarPanel('sharing-item')
     },
-    expandLinksPanel() {
-      this.setSidebarPanel('links-item')
```

Comment:
Same here, please use `sharing-item#linkShares`.

Reply:
This method is not used anymore, I just removed it